### DISCUSSION TITLE
Question regarding X79 boards and DSDT patching

### DISCUSSION THREAD

**Discussion by r-o-n-i** [xCuri0/ReBarUEFI#190] [General]
**System**
 - Motherboard: Not sure right now, some X79
 - BIOS Version: 2.15.1236
 - GPU: None so far
 - [ ] CSM is turned off. **Make sure to confirm this in the BIOS and not with GPU-Z or similar since it can be inaccurate**
 - [*] 4G decoding is enabled. **Make sure to confirm this in the BIOS and not with GPU-Z or similar since it can be inaccurate**
 - [ ] UEFIPatch is applied (see [Using UEFIPatch](https://github.com/xCuri0/ReBarUEFI/wiki/Using-UEFIPatch) for more information). On some motherboards [DSDT Patching](https://github.com/xCuri0/ReBarUEFI/wiki/DSDT-Patching) is also needed
 - [*] I have read [Common issues (and fixes)](https://github.com/xCuri0/ReBarUEFI/wiki/Common-issues-(and-fixes))

**Description**

In the Northbridge settings I can set MMIOH Size to maximum 128G. Does that mean that this (and system RAM and other devices) limits the total amount of possible VRAM? Is it possible that there is a hidden option for 256G or even higher? If yes, how to unlock? if no, is it possible to increase MMIOH Size by any other way?  Could the method described here 

https://github.com/xCuri0/ReBarUEFI/wiki/DSDT-Patching#x79-dsdt-patch 

help even if I did not apply any patch?

**Comment by xCuri0** [comment]
Why would you need such a large Mmioh size ? You would need to be running multiple GPUs with 32GB BAR to exceed it

**Reply by r-o-n-i** [reply]
I am planning to build a poor mans AI machine with as much VRAM as possible. If these boards are limited to 128GB I might regret the choice in the future. That's why I try to explore all options.

**Reply by xCuri0** [reply]
@r-o-n-i I think X79 will work fine with it when used along with patches

But to be safe I'd go with X99 which lets you change MMIO Size in the BIOS menu
